exporter/opentelemetry-exporter-jaeger/src/opentelemetry/exporter/jaeger/__init__.py

```diff
@@ -109,23 +100,23 @@ def __init__(
         service_name,
         agent_host_name=DEFAULT_AGENT_HOST_NAME,
         agent_port=DEFAULT_AGENT_PORT,
-        collector_host_name=None,
-        collector_port=None,
-        collector_endpoint=DEFAULT_COLLECTOR_ENDPOINT,
-        collector_protocol=DEFAULT_COLLECTOR_PROTOCOL,
+        collector_endpoint=None,
         username=None,
         password=None,
     ):
         self.service_name = service_name
-        self.agent_host_name = agent_host_name
-        self.agent_port = agent_port
+        self.agent_host_name = (
+            Configuration().EXPORTER_JAEGER_AGENT_HOST or agent_host_name
+        )
+        self.agent_port = (
+            Configuration().EXPORTER_JAEGER_AGENT_PORT or agent_port
+        )
         self._agent_client = None
-        self.collector_host_name = collector_host_name
-        self.collector_port = collector_port
-        self.collector_endpoint = collector_endpoint
-        self.collector_protocol = collector_protocol
-        self.username = username
-        self.password = password
+        self.collector_endpoint = (
+            Configuration().EXPORTER_JAEGER_ENDPOINT or collector_endpoint
+        )
+        self.username = username or Configuration().EXPORTER_JAEGER_USER
+        self.password = password or Configuration().EXPORTER_JAEGER_PASSWORD
```

Comment:
It looks like for username/password, precedence is given to the configured parameter, whereas for agent host,port and endpoint, the precedence is given to the environment variable. this would be confusing to me as a user. 

A good way to test this would be to ensure the precedence is consistent when both environment variables and the constructor is called with all parameters.